tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -774,3 +774,90 @@ Feature: Send a sharing invitations
       | Co Owner        | folder        | FolderToShare  |
       | Uploader        | folder        | FolderToShare  |
       | Manager         | folder        | FolderToShare  |
+
+
+  Scenario Outline: send share invitation to disabled user
+    Given user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    And the user "Admin" has disabled user "Brian"
+    When user "Alice" sends the following share invitation using the Graph API:
+      | resourceType    | <resource-type>    |
+      | resource        | <path>             |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    Then the HTTP status code should be "200"
```

Comment:
Note: We can share with disabled users. https://github.com/owncloud/ocis/issues/7962#issuecomment-1883179994